Makefile.PL

```diff
@@ -23,13 +23,30 @@ requires(
     'Zonemaster::LDNS'   => 2.0,
 );
 
+# Make all platforms include inc/Module/Install/External.pm
+requires_external_bin 'find';
+if ($^O eq "freebsd") {
+    requires_external_bin 'gmake';
+};
+
 sub MY::postamble {
-        return <<'MAKE_FRAG';
-	$(MYEXTLIB): share/Makefile
-	cd share && $(MAKE) all
-MAKE_FRAG
+    my $sharemakefile = 'share/GNUmakefile';
+    my $text;
+    if ($^O eq "freebsd") {
+        # Make FreeBSD use gmake for share
+        $text = q[GMAKE ?= "gmake"] . "\n"
+            . q[pure_all :: ] . "$sharemakefile\n"
+            . "\t" . q[cd share && $(GMAKE) all] . "\n";
```

Comment:
Could you make the string quoting consistent? At the very least locally, but preferably also with the rest of the repo. E.g. `q[]` isn't used anywhere else in zonemaster-engine. Also, mixing concatenation and interpolation only makes it harder to read.

Reply:
Updated, but no change of logic.

Reply:
The changed has been tested for both FreeBSD and Ubuntu, and passed.